src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Services/MetaGuidanceServiceTests.cs

```diff
@@ -45,32 +43,37 @@ public async Task Get()
             var releaseId = Guid.Parse("2ca4bbbc-e52d-4cb7-8dd2-541623973d68");
             const string releasePath = "publication/2016-17";
 
-            var blobStorageService = new Mock<IBlobStorageService>(MockBehavior.Strict);
+            var fileStorageService = new Mock<IFileStorageService>(MockBehavior.Strict);
             var metaGuidanceSubjectService = new Mock<IMetaGuidanceSubjectService>(MockBehavior.Strict);
 
-            var service = SetupMetaGuidanceService(blobStorageService: blobStorageService.Object,
+            var service = SetupMetaGuidanceService(
+                fileStorageService: fileStorageService.Object,
                 metaGuidanceSubjectService: metaGuidanceSubjectService.Object);
 
-            blobStorageService.Setup(
-                    mock => mock.DownloadBlobText(PublicContentContainerName, releasePath))
-                .ReturnsAsync(ReleaseJson);
+            fileStorageService.Setup(
+                    mock => mock.GetDeserialized<CachedReleaseViewModel>(releasePath))
+                .ReturnsAsync(new Either<ActionResult, CachedReleaseViewModel>(new CachedReleaseViewModel
```

Comment:
Won't it allow you to just return `new CachedReleaseViewModel` here?

Reply:
Yep it can, removed the wrapping `Either`